docs-partials/builder/azure/arm/SharedImageGalleryDestination-not-required.mdx

```diff
@@ -17,4 +17,6 @@
 
 - `specialized` (bool) - Set to true if publishing to a Specialized Gallery, this skips a call to set the build VM's OS state as Generalized
 
+- `shallow_replication` (bool) - Set to true if you want to publish using Shallow Replication, this is faster but skips copying the full image to the gallery image region, read more about Shallow Replication [here](https://learn.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries?tabs=azure-cli#shallow-replication)
```

Comment:
So this is a little confusing. At first I thought shallow replication meant that it would not wait for the replication to complete. But in reading the documentation. By setting this to true the image version is created but no replication occurs, as this is meant for testing purposes. 

I see the CLI has a flag called `replication-mode` which can be set to shallow. Would it be more explicit to cal call this attribute `skip_replication` or `use_shallow_replication`? 

```suggestion
- `use_shallow_replication` (bool) - Set to true to set the replication mode to shallow, which will publish the image version without replication. This option results in a faster build but skips the copying of the full gallery image to regions defined in `replication_regions`. Refer to [Shallow Replication](https://learn.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries?tabs=azure-cli#shallow-replication) for details on when to use shallow replication mode. 
```